Kernel/SortHelper.cpp

```diff
@@ -73,6 +73,14 @@ void SortHelper::getTypeSub(const Term* t, Substitution& subst)
   }  
 } // getTypeSub
 
+void SortHelper::getTypeArguments(TermList sort, TermStack& res)
```

Comment:
I am not understanding the name of this function. 

It seems it assumes sort is a sort and returns all its arguments. Is this really what "getTypeArguments" means? Could that be explained in a comment?

Reply:
Putting it like this, neither the name nor adding a function for this purpose make much sense. How about I inline this everywhere? It is only used on sorts in `Induction`, so the assertions are also kind of unnecessary.